projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_router.py

```diff
@@ -76,48 +77,31 @@ def test_router_send_tabular_data_for_ingestion_no_default_method(
 
 
 @patch(f"{IngesterRouter.__module__}.{IngesterBase.__name__}", spec=IngesterBase)
-def test_router_use_chained_ingest_plugins_send_object(
-    mock_ingester_base: MagicMock,
-):
-    router = create_ingester_router({"ingest_method_default": "test-ingest"})
-
-    router.add_ingester_factory_method(
-        "pre-ingest-test", lambda: MagicMock(spec=IIngesterPlugin)
-    )
-    router.add_ingester_factory_method(
```

Comment:
I like you had registered two infesters, then one is configured for usage via  `ingest_method_default`, 
and the verification is no chained infesters while multiple ones registered in the first place

Reply:
Expanded one of the tests to incorporate this suggestion. Thank you